rdflib/namespace/__init__.py

```diff
@@ -590,6 +590,34 @@ def compute_qname_strict(
 
             return self.__cache_strict[uri]
 
+    def expand_qname(self, curie: str) -> Union[str, None]:
+        """
+        Expand a qname (aka CURIE) of the form <prefix:element>, e.g. "rdf:type"
+        into its full expression:
+
+        >>> import rdflib
+        >>> g = rdflib.Graph()
+        >>> g.namespace_manager.expand_qname("rdf:type")
+        'http://www.w3.org/1999/02/22-rdf-syntax-ns#type'
+
+        Returns None if a namespace is not bound to the prefix or the prefix
+        is malformed.
```

Comment:
I think it is best to raise an error n such cases, the error can maybe be differentiated but for example if it is not a string I would raise a type error, if it does not contain ":" I would raise ValueError.

Reply:
Claro, implemented.